index.js

```diff
@@ -1,12 +1,22 @@
 let { readFileSync } = require("fs");
-let regex = /sourceMappingURL=data:application\/json;base64,(\S+)/;
+let findUrl = /sourceMappingURL=data:(\S+)/;
+let findMap = /;base64,(\S+)$/;
+
+function firstMatchedGroup(regex, data) {
+	let matches = regex.exec(data);
+	if(!matches) {
+		return null;
+	}
+	return matches[1];
+}
```

Comment:
FWIW, I like to put utility functions towards the bottom, reserving the top of a mdule for more significant bits. ( ❤️ function hoisting)